src/components/cards/ReleaseOverviewCard.vue

```diff
@@ -81,9 +69,24 @@ export interface Props {
 
 const { services } = defineProps<Props>();
 
-const formatDate = (dateString: string) => {
+const months = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December"
+];
+
+const formatDateFull = (dateString: string) => {
     const date = new Date(dateString);
-    return `${date.getDate()}-${date.getMonth() + 1}-${date.getFullYear()}`;
+    return `${date.getDate()} ${months[date.getMonth()]} ${date.getFullYear()}`;
```

Comment:
Is there a reason why we are not using the method `toLocaleString` of the `Date` class?  (See this SO-answer for an example: https://stackoverflow.com/a/18648314/11306008)

Reply:
wouldn't it be strange to have the month in your local language and the rest of the site in English?

Reply:
It is possible to pass "English" (or rather `en-GB`) to `toLocaleString` as an argument: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString.